frontend/public/components/masthead-toolbar.jsx

```diff
@@ -748,6 +773,13 @@ class MastheadToolbarContents_ extends React.Component {
           </ToolbarContent>
         </Toolbar>
         <AboutModal isOpen={showAboutModal} closeAboutModal={this._closeAboutModal} />
+        {this.state.reportBugLink ? (
+          <FeedbackModalLocalized
+            onOpenSupportCase={this.state.reportBugLink}
```

Comment:
Can we control the label in the model so it's not always "Open Support Case"? It would be better to use "Report a Bug" when we're opening a JIRA rather than an actual support case.